tools/ci_build/exclude_unused_ops.py

```diff
@@ -163,7 +209,7 @@ def should_exclude_op(domain, op_type, opset_from, opset_to=None):
 
         return True  # end of should_exclude_op(...)
 
-    def process_lines(lines, offset):
+    def process_lines(lines, offset, is_registration):
```

Comment:
>is_registration [](start = 37, length = 15)

Can "is_registration" be treated as an inner variable of process_lines instead of an argument, after all lines[offset] is the line we are looking at and we could tell if "is_registration" from there?

Reply:
Refactored so this doesn't apply anymore.

---
In reply to: [523935657](https://github.com/microsoft/onnxruntime/pull/5817#discussion_r523935657) [](ancestors = 523935657)